### PR TITLE
PRIM-240SecuritySettings(All Subtasks)

### PR DESCRIPTION
**Pull request by bhatt7821**
PRIM-240SecuritySettings(All Subtasks)

### CODE REVIEW

File: Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    [Binding]
+    public sealed class Add_Edit_Tenant_User_StepDefination
+    {
+        private IWebDriver _driver;
+
+        Actions act;
+
+        protected readonly PropertyReader _propertyReader;
+        protected readonly Add_Edit_Tenant_User_PageObjects _add_Edit_Tenant_User_PageObjects;
+        // protected string loggedUser;
```
Comment (ahaynes91) [inline Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs:25]: Remove please

File: Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        private readonly ScenarioContext _scenarioContext;
+
+        public Add_Edit_Tenant_User_StepDefination(IWebDriver driver)
+        {
+            _driver = driver;
+            _propertyReader = PropertyReader.Instance;
+            _add_Edit_Tenant_User_PageObjects = new Add_Edit_Tenant_User_PageObjects(_driver);
+            act = new Actions(driver);
```
Comment (ahaynes91) [inline Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs:33]: _driver

**Review by ahaynes91 — CHANGES_REQUESTED** [review]
Small changes suggested

File: Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs (hunk 1 above)
Comment (bhatt7821) [reply Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs:25]: ok

File: Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs (hunk 2 above)
Comment (bhatt7821) [reply Tests/Steps/Add_Edit_Tenant_User_StepDefination.cs:33]: _driver is an object of IWebDriver

**Comment by ahaynes91** [comment]
Changes were included in https://github.com/Affordability4You/QA_Automation/pull/8

**Pull request closed without merging**